ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -3015,6 +3015,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones-multisubnets-external-lb
+- as: vsphere-ipi-multi-vcenter-f28
```

Comment:
We also need a destructive ci job.

Reply:
ok. i will add it.

Reply:
Do we need to test it in 4.18 ?

Reply:
Good point. @WenXinWei pls also add them to 4.18.

Reply:
@liangxia it is done.